src/flow.jl

```diff
@@ -422,20 +427,18 @@ end
     to_river::Vector{T} | "m3 d-1"         # Part of subsurface flow [m³ d⁻¹] that flows to the river
     volume::Vector{T} | "m3"               # Subsurface volume [m³]
 
-    function LateralSSF{T}(args...) where {T}
+    function LateralSSF{T, Kh}(args...) where {T, Kh}
```

Comment:
These are the type parameters. Otherwise these should be inferred from the argument types and used in `new`. I think using a concrete type constructor is preferred.